src/command_modules/azure-cli-documentdb/azure/cli/command_modules/documentdb/_params.py

```diff
@@ -34,17 +33,17 @@ def validate_locations(ns):
         comps = item.split('=', 1)
         loc_dict.append(Location(location_name=comps[0], failover_priority=int(comps[1])))
     ns.locations = loc_dict
-    return ns
 
 register_cli_argument('documentdb', 'account_name', arg_type=name_type, help='Name of the DocumentDB Database Account', completer=get_resource_name_completion_list('Microsoft.DocumentDb/databaseAccounts'), id_part="name")
 
 register_cli_argument('documentdb regenerate-key', 'key_kind', **enum_choice_list(KeyKind))
 register_cli_argument('documentdb failover-priority-change', 'failover_policies', validator=validate_failover_policies, help="space separated failover policies in 'regionName=failoverPriority' format. E.g \"East US\"=0", nargs='+')
 
-register_cli_argument('documentdb create', 'resource_group_location', help="location of the resource group")
+register_cli_argument('documentdb create', 'account_name', completer=None)
+register_cli_argument('documentdb create', 'resource_group_name', help="name of the resource group")
+register_cli_argument('documentdb create', 'resource_group_location', ignore_type, help="location of the resource group")
 register_cli_argument('documentdb create', 'locations', validator=validate_locations, help="space separated locations in 'regionName=failoverPriority' format. E.g \"East US\"=0", nargs='+')
-register_cli_argument('documentdb create', 'default_consistency_level', help="default consistency level", **enum_choice_list(DefaultConsistencyLevel))
-register_cli_argument('documentdb create', 'resource_group_location', help="location of the resource group")
-register_cli_argument('documentdb create', 'max_staleness_prefix', help="maximum staleness prefix", type=int)
-register_cli_argument('documentdb create', 'max_interval_in_seconds', help="max_interval_in_seconds", type=int)
-register_cli_argument('documentdb create', 'ip_range_filter', help="IP Range Filter")
+register_cli_argument('documentdb create', 'default_consistency_level', help="default consistency level of the DocumentDB Database account", **enum_choice_list(DefaultConsistencyLevel))
+register_cli_argument('documentdb create', 'max_staleness_prefix', help="when used with Bounded Staleness consistency, this value represents the number of stale requests tolerated. Accepted range for this value is 1 - 2,147,483,647", type=int)
+register_cli_argument('documentdb create', 'max_interval', help="when used with Bounded Staleness consistency, this value represents the time amount of staleness (in seconds) tolerated. Accepted range for this value is 1 - 100", type=int)
```

Comment:
Since the parameter is still called "max_interval_in_seconds" (in custom.py) then the correct registration should be:
```Python
register_cli_argument('documentdb create', 'max_interval_in_seconds', options_list=('--max-interval',), help='...', type=int)
```
Otherwise, this alias will be applied to nothing.